src/reducers/notebook-reducer.js

```diff
@@ -103,6 +103,28 @@ const notebookReducer = (state = newNotebook(), action) => {
       return Object.assign({}, state, { jsmd, jsmdChunks });
     }
 
+    case "GETTING_NOTEBOOK_HISTORY": {
+      return Object.assign({}, state, {
+        gettingNotebookHistory: true
+      });
+    }
+
+    case "GOT_NOTEBOOK_HISTORY": {
+      const { notebookRevisionList } = action;
+      return Object.assign({}, state, {
+        gettingNotebookHistory: false,
+        notebookRevisionList
+      });
+    }
+
+    case "ERROR_GETTING_NOTEBOOK_HISTORY": {
+      return Object.assign({}, state, {
+        gettingNotebookHistory: false,
+        notebookRevisionList: undefined,
+        errorGettingNotebookHistory: true
```

Comment:
it looks like maybe this bit of state is not used anywhere, and it's not in the state schema -- maybe it's left over from something you tried out earlier?

Reply:
this was half-completed code for error handling. now done.